src/Neo/Ledger/MemoryPool.cs

```diff
@@ -659,5 +659,26 @@ internal bool ReVerifyTopUnverifiedTransactionsIfNeeded(int maxToVerify, DataCac
 
             return _unverifiedTransactions.Count > 0;
         }
+
+#if DEBUG
+        // This method is only for test purpose
+        // Do not remove it from the DEBUG build
+        internal void Clear()
```

Comment:
I think it doesn't hurt without debug, but it's ok for me

Reply:
If we compile in release maybe it fault because the UT require this method

Reply:
when add to test the same way as `#if DEBUG`

Reply:
@Jim8y we need to compile the whole sln in release, I think it won't work with this change